src/applications/mhv/secure-messaging/components/ComposeForm/FileInput.jsx

```diff
@@ -146,6 +146,7 @@ const FileInput = props => {
             }
             class="attach-file-button"
             data-testid="attach-file-button"
+            data-dd-action-name="Attach File Button"
```

Comment:
referencing my comment above, if we can use spaces instead of hyphens, I guess we would rather pick this option and be consistent across the app